docs/book/src/component-config-tutorial/tutorial.md

```diff
@@ -0,0 +1,33 @@
+# Tutorial: ComponentConfig
+
+Nearly every project that is built for Kubernetes will eventually need to
+support passing in additional configurations into the controller. These could
+be to enable better logging, turn on/off specific feature gates, set the sync
+period, or a myriad of other controls. Previously this was commonly done using
+cli `flags` that your `main.go` would parse to make them accessible within your
+program. While this _works_ it's not a future forward design and the Kubernetes
+community has been migrating the core components away from this and toward
+using versioned config files, referred to as "componentconfig".
```

Comment:
The phrase rather than the word is used throughout this tutorial
```suggestion
using versioned config files, referred to as "component configs".
```